tests/run.sh

```diff
@@ -18,7 +18,7 @@ exec &> >(tee -a "$CLUSTER_NAME.log")
 
 function destroy() {
   echo -e "\\e[34m Exiting... Destroying Tectonic...\\e[0m"
-  tectonic destroy --dir="${CLUSTER_NAME}"
+  $tectonic destroy --dir="${CLUSTER_NAME}"
```

Comment:
I'm surprised that ShellCheck is fine with this unquoted expansion.  Can you quote it with an uppercase variable name, or use `installer/tectonic` instead of the variable?  Using `installer/tectonic` would also give you the "copy/paste the tail end of the line to get `tectonic ...` for pasting into a shell" which may be your motivation for the lowercase variable.

Reply:
So back to how it was 3 iterations ago before aliases came to my mind. FWIW, aliases would've worked but with the caveat of having to define it before any function definition which makes use of the alias.